### DISCUSSION TITLE
Back button is not visible for a page in MAUI

### DISCUSSION THREAD

**Discussion by IXChinmayD** [PrismLibrary/Prism#3115] [.NET MAUI]
I am working on the Migration of Forms Application to .NET MAUI .NET 8 with Prism version 9.0.401-pre.  In Xamarin forms,

 I can see the navigation bar with the back button for some screens. But, in MAUI, the back button and title of the page are not visible.

I have attached screenshots for your reference

**In Xamarin Forms**

 ![Screenshot_20240402-150632](https://github.com/PrismLibrary/Prism/assets/155611333/3aa60a2b-7982-46c7-aaf3-5a170bfce093)

**MAUI**

 ![Screenshot 2024-04-02 150932](https://github.com/PrismLibrary/Prism/assets/155611333/dc1cadf4-38d8-4260-bdd4-558cffd73288)

**Comment by IXChinmayD** [comment]
The navbar is visible on Android when I write the below code:- 

`CreateWindow("NavigationPage/MainPage", exception => Console.WriteLine(exception))`

But it crashes on the iPhone with error: **No root window has been created**

**Reply by dansiegel** [reply]
The `No root window has been created` occurs because the navigation failed and thus no Window was created with a root page. If you look at the Navigation Exception it will give you hints as to what happened. The problem lies in your code.

**Comment by tommy-uytterhaegen** [comment]
Did you ever find a solution for this problem because I have the same problem :/
